src/keep_actions_alive.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+from github import Github
+import os
+import pprint
+import requests
+
+
+def get_parameters():
+    # Add more options for adding arguments here,
+    # such as CLI args, AWS Secrets Manager, config file, etc.
+
+    # Get keyword arguments from environment variables
+    return {
+        "login_or_token": os.getenv(
+            "GH_LOGIN_OR_TOKEN"
+        ),
+        "organization": os.getenv("GH_ORGANIZATION"),
+        "user": os.getenv("GH_USER"),
+    }
+
+
+if __name__ == "__main__":
+    pp = pprint.PrettyPrinter(indent=4)
+
+    parameters = get_parameters()
+
+    login_or_token = parameters["login_or_token"]
+    organization = parameters["organization"]
+    user = parameters["user"]
+
+    # Logic should be added here to support other authentication in the future
+    # `auth` is used by the requests package
+    auth = (user, login_or_token) if user else None
```

Comment:
If `user` is not present here we do a lot more work instead of terminating early on. As this will fail on line 50 when we do `if auth`.

This needs more testing, but from briefly looking at it. It does not seem that `user` really matters here, it can be anything. The personal access token is the only thing that actually does matter.

Reply:
If we can call `requests.put` without `user` then we can completely remove it for now.  I'm just looking for good documentation to show exactly what is acceptable by the auth argument in requests